service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -116,14 +115,13 @@ public ResponseEntity<RunSetListResponse> getRunSets(UUID methodId, Integer page
 
     if (methodId != null) {
       List<RunSet> filteredRunSet =
-          Collections.singletonList(
-              runSetDao.getRunSetWithMethodId(methodId, Optional.ofNullable(pageSize).orElse(10)));
+          Collections.singletonList(runSetDao.getRunSetWithMethodId(methodId));
```

Comment:
We only return latest run set associated with the method hence there is no limit to pass in page size here and it's associated DAO method.